Meadowlark-js/packages/meadowlark-core/src/model/DocumentIdentity.ts

```diff
@@ -5,6 +5,7 @@
 
 import { normalizeDescriptorSuffix } from '@edfi/metaed-core';
 import crypto from 'node:crypto';
+import { v4 as uuidv4 } from 'uuid';
```

Comment:
You don't need this. node:crypto has a UUID generator. https://nodejs.org/docs/latest-v16.x/api/crypto.html#cryptorandomuuidoptions